Core api for getting translation for a specified key for given locale (#705)
SAP/luigi: docs/luigi-core-api.md

**Meta**

-   **since**: 0.5.3

#### getTranslation

Gets translated text for the specified key in the current locale or in the specified one.

> Property values for token replacement in the localization key will be taken from the specified interpolations object.

Reviewer: ```suggestion
Property values for token replacement in the localization key are taken from the specified interpolations object.
```

##### Parameters

-   `key` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** key to be translated

> -   `interpolations` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** objects with properties that will be used for token replacements in the localization key

Reviewer: ```suggestion
-   `interpolations` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** objects with properties  used for token replacement in the localization key
```

> -   `locale` **locale** optional locale to get the translation for; default is the current locale

Reviewer: ```suggestion
-   `locale` **locale** optional locale for which you can get the translation. The default is the current locale.
```

**Meta**

-   **since**: 0.5.3

#### getTranslation

Gets translated text for the specified key in the current locale or in the specified one.
Property values for token replacement in the localization key are taken from the specified interpolations object.

##### Parameters

-   `key` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** key to be translated

> -   `interpolations` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** objects with properties  used for token replacement in the localization key

Reviewer: ```suggestion
-   `interpolations` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** objects with properties used for token replacement in the localization key
```
